docs/concepts/pdp/factdb.mdx

```diff
@@ -0,0 +1,230 @@
+---
+sidebar_position: 11
+title: Local Datastore (FactDB)
+---
+
+# PDP Local Datastore (FactDB)
+
+FactDB is an embedded datastore to manage larges data volumes inside the PDP.
```

Comment:
Add link on the PDP word

Reply:
Well, it is inside of the PDP subject, do we really need to link to its parent?